### ISSUE TITLE
Euros a problem

### ISSUE THREAD

**Issue by andersonimes** [juftin/lunchable-primelunch#7]
Hello! I am getting the following error trying to process a file. I'm wondering if maybe this isn't designed to process a file with Euros, maybe? Here is the error:

> ValueError: could not convert string to float: '€2656'

<img width="752" alt="Screenshot 2024-12-10 at 1 43 44 PM" src="https://github.com/user-attachments/assets/1a52f9df-8252-472d-8dec-13da23725b36">

**Comment by andersonimes** [comment]
Oh. I think I know why. The values are all denoted with commas for decimals.

"€26,56"

Can you fix it or should I just regex the files before using the tool?

**Comment by andersonimes** [comment]
Nope - that did not fix it.

ValueError: could not convert string to float: '€26.56'

Must be the euro symbol maybe, but looking at the code, there appears to be currency matching going on so I don't think I can just change it to a dollar symbol or remove it without impacting something else.

**Comment by juftin** [comment]
Let me know if this didn't resolve the issue 🤝  https://github.com/juftin/lunchable-primelunch/blob/d2eab76fa3e605811a99ad02c03662737fead687/lunchable_primelunch/primelunch.py#L220-L222

**Comment by andersonimes** [comment]
![result](https://github.com/user-attachments/assets/3be24a67-76c8-4d9e-afaa-3c61e51dcc8a)
Pretty much the same result.

**Comment by juftin** [comment]
@andersonimes could you share a CSV with a few test records so I can debug on my end?

**Comment by andersonimes** [comment]
I hope I don't regret this :) 

[amazon_order_history.csv](https://github.com/user-attachments/files/18219940/amazon_order_history.csv)

**Comment by andersonimes** [comment]
Hello! Did you get a chance to take a look at this?

**Comment by juftin** [comment]
Yup, thanks for sending that! This was a tough fix but I believe I've got it. If you run into any issues try setting the `LC_ALL=<your locale code>` environment variable.

**Comment by juftin** [comment]
Let me know how the new version works 😅

**Comment by andersonimes** [comment]
This did not resolve the problem.

![Image](https://github.com/user-attachments/assets/800ccf2e-d6b0-433b-a3ea-cbdcaf083678)

**Comment by andersonimes** [comment]
@juftin sorry to say I think you should reopen this.
